foxpuppet/tests/test_browser_model.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from foxpuppet import FoxPuppet
+
+
+class TestBrowserModel(object):
+
+    def test_new_private_window(self, selenium):
+        """Tests opening a new private browsing window via menu"""
+        foxpuppet = FoxPuppet(selenium)
+        with foxpuppet.windows.wait_for_new_window():
```

Comment:
@silne30 Ahh, okay. Dave had said something about context manager merged into Selenium. I mixed them up. I will add them in the appropriate places though.